pkg/kubelet/eviction/helpers.go

```diff
@@ -27,6 +27,7 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	"k8s.io/klog"
+	v1resource "k8s.io/kubernetes/pkg/api/v1/resource"
```

Comment:
@tallclair - my apologies - it wasn't 100% clear to me if anything should be rectified in kubernetes/pkg/api/resource as well (the helpers.go looks the same as what's in v1, but out of date).  Some of the versioning API is still a bit magical to me; can you clarify?

Reply:
https://github.com/kubernetes/kubernetes/pull/81104 :smile:

Reply:
This was very triggering to me. Look what you did: https://github.com/kubernetes/kubernetes/pull/81189
https://github.com/kubernetes/kubernetes/pull/81206